TestHelpers.Tests/MockFileReadAllBytesTests.cs

```diff
@@ -47,5 +47,15 @@ public void MockFile_ReadAllBytes_ShouldThrowFileNotFoundExceptionIfFileDoesNotE
 
             Assert.Throws<FileNotFoundException>(action);
         }
+
+        [Test]
+        public void MockFile_ReadAllBytes_ShouldTolerateAltDirectorySeparatorInPath()
+        {
+            var fileSystem = new MockFileSystem();
+
+            fileSystem.AddFile(XFS.Path(@"C:\test.txt"), new MockFileData("content"));
+
+            Assert.AreEqual("content", fileSystem.File.ReadAllText(XFS.Path("C:/test.txt")));
```

Comment:
The name of the test is ReadAllBytes, but we're calling ReadAllText. It also looks like were hardcoding the forward slash. If AltDirectorySeparator were to change, this test might not make anymore sense. If you want to specifically test forward slash, I'd probably prefer calling out the way that were honestly just testing forwards and backwards slashes.

Reply:
Used byte array instead and built paths using `fileSystem.Path.DirectorySeparatorChar` and `.AltDirectorySeparatorChar`.